Header header
int32 index
int32 value_type
float32 value
